tasks/main.yml

```diff
@@ -22,12 +14,21 @@
     - ansible_os_family == "Debian"
 
 - name: Install pip (AmazonLinux)
-  package:
-    name:
-      - python27-pip
-      - python27-devel
-      - python36-pip
-      - python36-devel
+  block:
+    # This is required temporarily, because of
+    # https://github.com/ansible/ansible/issues/56583
+    - name: Temporary workaround for ansible/ansible#56583
+      set_fact:
+        ansible_facts:
+          pkg_mgr: yum
+
```

Comment:
Let's remove this blank line, just so the block is visually a contiguous unit.

Reply:
Sure, I wasn't sure which would be preferred.